deployments/go.mod

```diff
@@ -10,27 +10,61 @@ replace (
 
 require (
 	github.com/clarketm/json v1.17.1
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorHostPolicy v0.0.0-00010101000000-000000000000
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorPolicy v0.0.0-00010101000000-000000000000
+	github.com/kubearmor/KubeArmor/pkg/KubeArmorHostPolicy v0.0.0-20220128051912-b9f5851b939b
```

Comment:
let's use replace() not to depend on the specific version like [go.mod](https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/go.mod)

Reply:
We still have the replace directive but I specified version number regardless of that because importing this package with the v000 errored out with 
>invalid version: unknown revision 0000000

Reply:
Is this error popped up because you didn't specify all required paths?
```
replace (
	github.com/kubearmor/KubeArmor => ../
	github.com/kubearmor/KubeArmor/deployments => ./
	github.com/kubearmor/KubeArmor/deployments/get => ./get
	github.com/kubearmor/KubeArmor/pkg => ../pkg
	github.com/kubearmor/KubeArmor/pkg/KubeArmorPolicy => ../pkg/KubeArmorPolicy
	github.com/kubearmor/KubeArmor/pkg/KubeArmorHostPolicy => ../pkg/KubeArmorHostPolicy
)
```

I don't think that defining the specific version of our code will raise unexpected side-effects (e.g., difficulty in debugging, additional modification when we have some changes, unintended errors when some other guys make code changes because they don't expect that they also need to update this file) later.

Reply:
When we use replace directive it disregards the version specified in require directive for local module development.

I cannot add these replace driectives to kArmor since it is an independent tool. Perhaps if someone else creates a new utility for KubeArmor they cannot leverage the replace directive as well.

go 1.18 beta 2 introduced a new mode called workspace mode to handle these well but I am not sure what's expected to be done right now https://go.googlesource.com/proposal/+/master/design/45713-workspace.md